sound/soc/sof/pcm.c

```diff
@@ -298,11 +298,11 @@ static int sof_pcm_trigger(struct snd_soc_component *component,
 
 	switch (cmd) {
 	case SNDRV_PCM_TRIGGER_PAUSE_PUSH:
-		ipc_first = true;
```

Comment:
commit message: if it's a fix, do we want a Fixes tag? Otherwise it's better to reword along the lines of "improve IPC4 trigger sequence"